code/starfield/starfield.cpp

```diff
@@ -181,6 +182,24 @@ int Num_debris_nebula = 0;
 bool Dynamic_environment = false;
 bool Motion_debris_override = false;
 
+static int setMotionDebris(sexp::SEXPParameterExtractor* extractor) {
+	Motion_debris_override = extractor->getArg<bool>();
+	return SEXP_TRUE;
+}
+
+const auto __UNUSED SetMotionDebrisSexp =
+	sexp::EngineSEXP::create("set-motion-debris-override")
+		.helpText("Controls whether or not motion debris should be active.\r\n"
+				  "\tThis overrides any choice made by the user through the -nomotiondebris commandline flag.")
+		.category(OP_CATEGORY_CHANGE2)
+		.subcategory(CHANGE_SUBCATEGORY_CUTSCENES)
+		.returnType(OPR_NULL)
+		.beginArgList()
+		.arg(OPF_BOOL, "Sets the new state of the motion debris")
```

Comment:
Add the boolean information (true to disable, false to enable) to the help text, since that's counterintuitive.

Reply:
Fixed.